staging/vhost-device-sound/src/stream.rs

```diff
@@ -250,11 +250,11 @@ impl std::fmt::Debug for Buffer {
 }
 
 impl Buffer {
-    pub fn new(descriptor: virtio_queue::Descriptor, message: Arc<IOMessage>) -> Self {
+    pub fn new(data_descriptor: virtio_queue::Descriptor, message: Arc<IOMessage>) -> Self {
```

Comment:
Sorry, I'm bit confused, maybe there is something strange caused by rebasing but in this PR we change this field from `data_descriptor: virtio_queue::Descriptor` to `bytes: Vec<u8>`  in the 2nd patch, then in `descriptor: virtio_queue::Descriptor` in the 4th and now back to `data_descriptor: virtio_queue::Descriptor` .

Can you check what is going on?

Reply:
Will check, sounds like a rebase screw up